src/material/card/_card-theme.scss

```diff
@@ -1,70 +1,80 @@
-@use '../core/theming/theming';
+@use 'sass:map';
 @use '../core/mdc-helpers/mdc-helpers';
+@use '../core/theming/theming';
 @use '../core/style/private';
 @use '../core/typography/typography';
+@use '../core/tokens/m2/mdc/elevated-card' as m2-mdc-elevated-card;
+@use '../core/tokens/m2/mdc/outlined-card' as m2-mdc-outlined-card;
 @use '@material/card/elevated-card-theme' as mdc-elevated-card-theme;
 @use '@material/card/outlined-card-theme' as mdc-outlined-card-theme;
 @use '@material/typography' as mdc-typography;
-@use '@material/theme/theme-color' as mdc-theme-color;
-@use 'sass:color';
-@use 'sass:map';
-@use 'sass:meta';
 
 @mixin color($config-or-theme) {
   $config: theming.get-color-config($config-or-theme);
-  $foreground: map.get($config, foreground);
+  $mdc-elevated-card-color-tokens: m2-mdc-elevated-card.get-color-tokens($config);
+  $mdc-outlined-card-color-tokens: m2-mdc-outlined-card.get-color-tokens($config);
 
-  @include mdc-helpers.using-mdc-theme($config) {
-    $on-surface: mdc-theme-color.prop-value(on-surface);
-    $surface: mdc-theme-color.prop-value(surface);
+  // Add values for MDC card tokens.
+  .mat-mdc-card {
+    @include mdc-elevated-card-theme.theme($mdc-elevated-card-color-tokens);
+    @include mdc-outlined-card-theme.theme($mdc-outlined-card-color-tokens);
 
-    .mat-mdc-card {
-      // MDC's theme has `container-elevation` and `container-shadow-color` tokens, but we can't
-      // use them because they output under a `.mdc-card` selector whereas the rest of the theme
-      // isn't under any selector. Even if the mixin is pulled out of the selector, it throws a
-      // different error.
-      @include private.private-theme-elevation(1, $config);
-      @include mdc-elevated-card-theme.theme((
-        container-color: $surface,
-      ));
-    }
+    // TODO(mmalerba): This should be based on tokens, but the elevation tokens don't seem to be
+    //  working currently. They wind up setting values like `box-shadow: var(--e); --e: 1`
+    @include private.private-theme-elevation(1, $config);
```

Comment:
While we're messing around with the specificity of the card, should we make sure that this use case works as well? https://github.com/angular/components/issues/26094

Reply:
We should fix it, but I'd like to make these token PRs as easy as possible to land, which means not trying to fix bugs in the same PR.